portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,10 +39,13 @@ public void init() {
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = getParameter(request, "text-input");
-    
     // want to display the text as an ArrayList
     comments.add(text);
     response.setContentType("text/html;");
```

Comment:
Sorry should've probably brought this up in your last PR, but did you want to write anything in the post response? You're setting the contentType but not writing anything like `response.getWriter().println(text);`

Reply:
correct me if i'm wrong, but i think that by redirecting to /index.html, it proceeds to call the get() function, which prints out everything. 

i think a redirect is ok but probably slower than just doing `response.getWriter().println();`